projects/kubernetes-sigs/image-builder/patches/0004-Ubuntu-22.04-support-and-improvements.patch

```diff
@@ -64,86 +59,6 @@ index 143c83999..152d33da6 100644
  validate-raw-rhel-8: ## Validates RHEL 8 RAW image packer config
  validate-raw-all: $(RAW_VALIDATE_TARGETS) ## Validates all RAW Packer config
  
-diff --git a/images/capi/packer/ova/ubuntu-2204-efi.json b/images/capi/packer/ova/ubuntu-2204-efi.json
```

Comment:
These patches are getting removed as this change has already been [upstreamed](https://github.com/kubernetes-sigs/image-builder/commit/e8e582deeb77945a34eed2e5c95154d0c14275b0).